### PR TITLE
Update dlp-conditions-and-exceptions.md

### PR DESCRIPTION
**Pull request by aamalhot**
add AccessScope parameter

### CODE REVIEW

**Comment by aamalhot** [comment]
@chrfox to review

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 1]
```diff
@@ -98,7 +99,7 @@ To configure the sender address location at a DLP rule level, the parameter is *
 |Content contains|condition: *ContentContainsSensitiveInformation* <br/> exception *ExceptIfContentContainsSensitiveInformation*|SensitiveInformationTypes|Messages or documents that contain sensitive information as defined by Microsoft Purview Data Loss Prevention (DLP) policies.|
 |Subject or Body matches pattern|condition: *SubjectOrBodyMatchesPatterns* <br/> exception: *ExceptIfSubjectOrBodyMatchesPatterns*|Patterns|Messages where the subject field or message body contains text patterns that match the specified regular expressions.|
 |Subject or Body contains words|condition: *SubjectOrBodyContainsWords* <br/> exception: *ExceptIfSubjectOrBodyContainsWords*|Words|Messages that have the specified words in the subject field or message body|
-|
+
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md:102]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @aamalhot, kindly commit the suggested change to remove the additional blank line. Many thanks.

**Comment by ybhargav1995** [comment]
Approved

**Pull request merged**